index.html

```diff
@@ -206,6 +206,23 @@ <h3>Skills</h3>
                     </div>
                 </div>
             </section>
+            <section class="contact">
+                <div class="text-primary mod-text">
+                    <h1 class="primary-one zero center-white">Contact me</h1>
+                    <p class="contact-primary-two center-white">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                      <br>  I’d love to help with it!</p>
+                </div>
+                <div class="from-container">
+                    <form>
+                        <input type="text" name="name" id="name" placeholder="Lorem Ipsum">
```

Comment:
- [ ] As specified in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) please make sure you're using the  `maxlength="30"` attribute on the name input as this will limit the maximum number of characters on your input. ✔
    - Here's the requirement as a reminder:
     ![image](https://user-images.githubusercontent.com/104968236/230116144-e4bdcd90-e33d-4dde-869d-1dca44280ada.png)

- [ ] Also kindly make sure you're using the `required` attribute on **all of your inputs** as this will ensure the user doesn't accidentally submit an empty value.